clients/web/src/lib/components/atoms/form/Textarea/Textarea.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+    import {slide} from "svelte/transition";
+    import {nanoid} from "nanoid";
+
+    import type {FormControlSize, FormControlState} from "../form.types";
+
+    export let size: FormControlSize = "md";
+    export let label: string;
+    export let placeholder: string | undefined = undefined;
+    export let disabled: boolean = false;
+    export let state: FormControlState = "none";
+    export let error: string | undefined = undefined;
+
+    const labelId = `textarea_${nanoid()}`;
+    const errorId = `textarea-error_${nanoid()}`;
+
+    let rows: number;
+    $: {
```

Comment:
Would a lookup object be easier to grok here?

Reply:
I like the explicit-ness and purpose of a switch statement here